src/status_im/ui/screens/discover/components/views.cljs

```diff
@@ -79,7 +80,8 @@
           [react/text {:style           styles/discover-list-item-name
                        :font            :medium
                        :number-of-lines 1}
-           (display-name me? account-name contact-name name whisper-id)]]]]
+           (cond-> (display-name me? account-name contact-name name whisper-id)
```

Comment:
if display-name is used only in these 2 places then this logic should be in display-name as well instead of duplicated